admin_manual/installation/deployment_recommendations.rst

```diff
@@ -3,299 +3,363 @@ Deployment Recommendations
-   application servers. 
+Load Balancer
+~~~~~~~~~~~~~
 
-* Database
-   MySQL/MariaDB Galera cluster with master-master replication. (InnoDB storage engine, MyISAM is not supported, see: :ref:`db-storage-engine-label`)
+HAProxy running on a dedicated server in front of the application servers. 
+Sticky session needs to be used because of local session management on the application servers. 
 
-* Backup
-   Minimum daily backup without downtime. All MySQL/MariaDB statements should 
-   be replicated to a backup MySQL/MariaDB slave instance.
-   
-    * Create a snapshot on the NFS storage server. 
-    * At the same time stop the MySQL replication.
-    * Create a MySQL dump of the backup slave.
-    * Push the NFS snapshot to the backup.
-    * Push the MySQL dump to the backup.
-    * Delete the NFS snapshot.
-    * Restart MySQL replication.
-
-* Authentication
-   User authentication via one or several LDAP or Active Directory servers. 
-   (See `User Authentication with LDAP`_  for information on configuring 
-   ownCloud to use LDAP and AD.)
-   
-* LDAP 
-   Read-only slaves should be deployed on every application server for 
-   optimal scalability
-
-* Session Management
-   Session management on the application server. PHP sessions are stored 
-   in a tmpfs mounted at the operating system-specific session storage 
-   location. You can find out where that is by running ``grep -R 
-   'session.save_path' /etc/php5`` and then add it to the ``/etc/fstab`` file, 
-   for example: 
-   ``echo "tmpfs /var/lib/php5/pool-www tmpfs defaults,noatime,mode=1777 0 0" 
-   >> /etc/fstab``.
-
-* Memory Caching
-   A memcache speeds up server performance, and ownCloud supports four 
-   memcaches; refer to `Configuring Memory Caching`_ for information on 
-   selecting and configuring a memcache.
+Database
+~~~~~~~~
+
+MySQL/MariaDB Galera cluster with `master-master replication`_. 
+InnoDB storage engine, MyISAM is not supported, see: :ref:`db-storage-engine-label`.
+
+Backup
+~~~~~~
+
+Minimum daily backup without downtime. 
+All MySQL/MariaDB statements should be replicated to a backup MySQL/MariaDB slave instance.
    
-* Storage
-   Use an off-the-shelf NFS solution, such as IBM Elastic Storage or RedHat 
-   Ceph.
+- Create a snapshot on the NFS storage server. 
+- At the same time stop the MySQL replication.
+- Create a MySQL dump of the backup slave.
+- Push the NFS snapshot to the backup.
+- Push the MySQL dump to the backup.
+- Delete the NFS snapshot.
+- Restart MySQL replication.
+
+Authentication
+~~~~~~~~~~~~~~
+
+User authentication via one or several LDAP or Active Directory servers. 
+See `User Authentication with LDAP`_  for information on configuring ownCloud to use LDAP and AD.
    
-* ownCloud Edition
-   Enterprise Edition. (See `ownCloud Server or Enterprise Edition`_ for 
-   comparisons of the ownCloud editions.)
+LDAP 
+~~~~~
+
+Read-only slaves should be deployed on every application server for optimal scalability.
```

Comment:
Thanks. FWIW, this PR is largely a reorganisation of the original content. I'll amend it to remove this section.